src/components/core/testing/a11y-tree-snapshot.ts

```diff
@@ -22,10 +25,16 @@ async function a11yTreeEqualSnapshot(): Promise<void> {
 /**
  * The function creates and tests the accessibility tree snapshot on each browser.
  * If a template is passed, it will be instantiated before the snapshot is taken.
+ * Note:
+ * We skip a11yTreeSnashots in debug environment because they not consistent on Puppeteer
  * @param title The title of the section
  * @param template The optional html template
  */
-export function testA11yTreeSnapshot(title = 'A11y tree', template?: TemplateResult): void {
+export function testA11yTreeSnapshot(
+  template?: TemplateResult,
+  title = 'A11y tree',
+  skip: { chrome?: boolean; firefox?: boolean; safari?: boolean } = {},
```

Comment:
Question: the skip param is used only twice. Is it worth it, or can we manage the functionality somehow in the two specific spec files?

Reply:
Since the `testA11yTreeSnapshot` function is seen as a black box from the components, we don't have a way to influence its behavior if not by using some params :/

Reply:
Got it 👍